rho/facts.py

```diff
@@ -168,6 +168,9 @@ def new_fact(fact_name, description, is_default=None,
 new_fact('jboss.eap.installed-versions',
          'List of installed versions of JBoss EAP',
          is_default=False)
+new_fact('jboss.eap.locate-jboss-modules-jar',
+         'Use locate to find jboss-modules.jar',
+         is_default=True)
```

Comment:
Should this and the ones above be added to the JBOSS_FACTS category?

Reply:
Added locate to the JBOSS_FACTS category, thank you!

Everything in JBOSS_FACTS will be run when the user says `--facts jboss`, so I think we don't want to include the find-based commands in that. We should probably find a better name for the fact group, but I don't have one right now.